lang/lua-bit32/Makefile

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (C) 2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lua-bit32
+PKG_VERSION:=5.3.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_URL:=git://github.com/keplerproject/lua-compat-5.2.git
+PKG_SOURCE_VERSION:=bitlib-5.3.0
+
+PKG_MAINTAINER:=Alex R <alex@totalwebservices.net>
+PKG_LICENSE=MIT
+
+LUA_MODULE_PATH:=/usr/lib/lua
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/lua-bit32
+  SUBMENU:=Lua
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=lua-bit32
+  URL:=http://www.lua.org/manual/5.2/manual.html#6.7
+  DEPENDS:= +lua
+endef
+
+define Package/lua-bit32/description
+   bit32 is the native Lua 5.2 bit manipulation library, in the version from Lua 5.3; it is compatible with Lua 5.1, 5.2 and 5.3.
+endef
+
+TARGET_CFLAGS += $(FPIC) -shared
+TARGET_LDFLAGS += -llua
+
+LUA_MODULE_PATH:=/usr/lib/lua
+
+define Build/Compile
+	$(TARGET_CROSS)gcc $(TARGET_CFLAGS) $(TARGET_LDFLAGS) -I $(PKG_BUILD_DIR)/c-api/ -o $(PKG_BUILD_DIR)/bit32.so $(PKG_BUILD_DIR)/lbitlib.c
```

Comment:
Is upstream _really_ that broken that you have to compile it yourself? Have you considered just using lua-bitop? it's already packaged.